### DISCUSSION TITLE
fine tuning whisper for translation task

### DISCUSSION THREAD

**Discussion by sutungpo** [openai/whisper#2067] [Q&A]
huggingface has already fine tuning about whisper[https://huggingface.co/blog/fine-tune-whisper], but it's transcription task, anyone knows how to fine tune the translation task?

**Comment by Devansh-Shah11** [comment]
Replace task='transcribe' to task='translate'
As it only translates to English, we can remove the language parameter as well!

Hope it helps :)

**Reply by sutungpo** [reply]
no, I think it will not work, at least the evaluation metrics should be changed, anyone has the good experience for fine tuning translation task?

**Reply by Devansh-Shah11** [reply]
Yes correct,
also change the evaluation metrics to BLEU score 
Reference: https://huggingface.co/spaces/evaluate-metric/bleu
